lib/templates/slim/scaffold/_form.html.slim

```diff
@@ -0,0 +1,8 @@
+= semantic_form_for @<%= singular_name %> do |f|
+  = f.inputs do
+  <%- attributes.each do |attribute| -%>
+    = f.input :<%= attribute.name %>
+  <%- end -%>
+
+  = f.actions do
+    = f.action :submit, as: :input
```

Comment:
What is this for? is it needed?

Reply:
Even I was not sure that if we really need this but pushed it to get your views

Reply:
this was just to show as an example how form will look like and not needed so removed